### PR TITLE
Add warning/flag if segmentation needs to be modified

### PR DESCRIPTION
**Pull request by alexfoias**
This pull request aims to fix [issue 42](https://github.com/neuropoly/sct_testing_management/issues/42)

### CODE REVIEW

File: sct_annotation/annotations/migrations/0014_labeledimage_label_needs_correction.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Generated by Django 1.11.14 on 2018-07-30 16:06
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('annotations', '0013_auto_20180417_1519'),
```
Comment (jcohenadad) [inline sct_annotation/annotations/migrations/0014_labeledimage_label_needs_correction.py]: that is `0013_auto_20180417_1519`? comments would be welcome

**Comment by alexfoias** [comment]
@jcohenadad  I did a clone of this branch under joplin. To access the simulated server use the ssh tunnelling explained in : https://github.com/neuropoly/sct_testing_management#development. Once you do the tunnelling, open the [link](http://localhost:8100/admin/annotations/image/?) in your internet browser. Username and password information available [here](https://www.neuro.polymtl.ca/internal_resources/list_of_computers?&#joplin_64-core_cpu)

**Comment by alexfoias** [comment]
1. In your terminal run : 
`ssh -L 8100:joplin.neuro.polymtl.ca:8000 neuropoly_local@joplin.neuro.polymtl.ca`. Use the password for local neuropoly account. Do not close the terminal window.

2. In your web browser go to : [link](http://localhost:8100/admin/annotations/image/?). Use username: sct_test_user; passwd: management.

3. On the lower right side of the page you should see the filter by label needs correction. If you edit one of the images, a checkbox should be present in the labeled images section.

**Comment by jcohenadad** [comment]
@alexfoias fantastic!!! great job 👍 
could you pls add [this explanation](https://github.com/neuropoly/sct_testing_management/pull/50#issuecomment-410073343) under new section: "How to test the DB from a specific branch".
and i assume that you already configured neuropoly_local to be in-tuned with this branch, so could you pls also give explanation as to how to update to a specific branch (with git checkout), thx

**Comment by alexfoias** [comment]
@jcohenadad I created the wiki page [here](https://github.com/neuropoly/sct_testing_management/wiki/Development)

**Pull request merged**